molecule/default/nc23-converge.yml

```diff
@@ -1,18 +1,18 @@
----
```

Comment:
Excellent you can remove this file then.

Reply:
Done :)